package.json

```diff
@@ -5,6 +5,7 @@
   "description": "A Fast Rust-based web bundler",
   "private": true,
   "scripts": {
+    "preinstall": "node scripts/meta/setup.mjs",
```

Comment:
We already have `pnpm:devPreinstall` due to `preinstall` is actually running after `pnpm install.`